src/metabase/driver/sql_jdbc/execute.clj

```diff
@@ -133,6 +135,15 @@
   ^DataSource [database]
   (:datasource (sql-jdbc.conn/db->pooled-connection-spec database)))
```

Comment:
Is it safe to assume this will always return an instance of `PoolBackedDataSource` ?

Reply:
@camsaul , could use your eyes on this one; thanks.